src/ui/QuerySuggestPreview/QuerySuggestPreview.ts

```diff
@@ -22,6 +22,20 @@ export interface IQuerySuggestPreview {
   executeQueryDelay?: number;
 }
 
+/**
+ * This component renders previews of the top query results matching the currently focused query suggestion in the search box.
+ *
+ * As such, this component only works when the search interface can
+ * [provide Coveo Machine Learning query suggestions](https://docs.coveo.com/en/340/#providing-coveo-machine-learning-query-suggestions).
+ *
+ * **Example**
```

Comment:
I'd personally replace the whole **Example** with:

```
This component should be initialized on a `div` which can be nested anywhere inside the root element of your search interface.
```

Reply:
Alright, replacing it.